### PR TITLE
release/v5.1.0

### LINKED ISSUE #478: Add TCGPlayer SKUS file docs
in downloads, and abstract models

### LINKED ISSUE #479: Add Historic to Legalities data model
It would be nice to add Historic as a format in the legalities data model. Historic Brawl may be nice too, as it has a slightly different ban list. Scryfall supports Historic, though it does not support Frontier like MTGJSON does.

https://mtgjson.com/data-models/legalities/

Historic also adds the concept of a suspended card, which is new.
See https://magic.wizards.com/en/articles/archive/news/march-9-2020-banned-and-restricted-announcement

### LINKED ISSUE #480: keyruneCode link is outdated
On the (at least, from what I noticed) following page, the link in the keyruneCode documentation is outdated:
https://mtgjson.com/data-models/set/

I believe the website has been updated to be https://keyrune.andrewgioia.com/
according to its github README (https://github.com/andrewgioia/keyrune)

### LINKED ISSUE #482: V5.0.0 changelog change
v5 initial changelog should indicate we dropped `mtgstocksId` because the data they provide is now duplicated by our other entities.

### PR DESCRIPTION
**Pull request by staghouse**
Fix #478 
Fix #479 
Fix #480 
Fix #482

### CODE REVIEW

File: docs/.vuepress/src/schemas/compiledList.schema.json [hunk 1]
```diff
@@ -137,6 +137,12 @@
     "description": "Files containing all cards, restricted to cards legal in the Standard format.",
     "introduced": "v4.6.0"
   },
+  "TcgplayerSkus": {
+    "type": "string",
+    "example" : "\"TcgplayerSkus\"",
+    "description": "Files containing {{link@TCGplayer@https://tcgplayer.com}} SKU information based on card UUIDs.",
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/compiledList.schema.json:143]: Should include referral link (can be found on homepage)

File: docs/.vuepress/src/schemas/card.schema.json [hunk 2]
```diff
@@ -402,6 +403,16 @@
...
   },
+  "originalReleaseDate": {
+    "type": "string",
+    "example": "\"2020-11-29\"",
+    "description": "Release date in {{link@ISO 8601@https://www.iso.org/iso-8601-date-and-time-format.html}} format for the card.",
+    "isManifestProperty": true,
+    "attributes": [
+  	  "optional"
+  	],
+    "introduced": "v5.0.2"
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/card.schema.json:414]: v5.1.0

File: docs/.vuepress/src/schemas/deck.schema.json [hunk 3]
```diff
@@ -6,6 +6,13 @@
     "isManifestProperty": true,
     "introduced": "v4.3.0"
   },
+  "commander": {
+    "type": "object",
+    "example": "{{link@Card@/data-models/card/}}",
+    "description": "The card that is the Commander in this deck. See the {{link@Card@/data-models/card/}} data model.",
+    "attributes": [ "optional" ],
+    "introduced": "v5.0.2"
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/deck.schema.json:14]: v5.1.0

File: docs/.vuepress/src/schemas/legalities.schema.json [hunk 4]
```diff
@@ -44,6 +44,15 @@
       "optional"
     ]
   },
+  "historic": {
+    "type": "string",
+    "example": "",
+    "description": "Sol Ring is not legal in the Historic play format. This property is omitted.",
+    "introduced": "v5.0.2",
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/legalities.schema.json:51]: v5.1.0

File: docs/.vuepress/src/schemas/meta.schema.json [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
     "type": "string",
-    "example": "\"2020-07-06\"",
+    "example": "\"2020-12-02\"",
     "description": "{{link@ISO 8601@https://www.iso.org/iso-8601-date-and-time-format.html}} date of the MTGJSON build.",
     "introduced": "v4.0.0"
   },
   "version": {
     "type": "string",
-    "example": "\"5.0.0+20200202\"",
+    "example": "\"5.0.2+20201202\"",
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/meta.schema.json:10]: v5.1.0+20200118

File: docs/.vuepress/src/schemas/set.schema.json [hunk 6]
```diff
@@ -100,22 +105,35 @@
...
     "introduced": "v4.3.2"
   },
-  "mcmName": {
-    "type": "string",
-    "example": "\"Ravnica Allegiance\"",
-    "description": "The Magic Card Market set name.",
+  "mcmId": {
+    "type": "integer",
+    "example": "2411",
+    "description": "The {{link@Magic Card Market@https://www.cardmarket.com/en/Magic}} set ID.",
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/set.schema.json:115]: include referral

File: docs/.vuepress/src/schemas/set.schema.json [hunk 7]
```diff
@@ -100,22 +105,35 @@
...
     ],
     "introduced": "v4.4.0"
   },
-  "mcmId": {
+  "mcmIdExtras": {
     "type": "integer",
-    "example": "2411",
-    "description": "The Magic Card Market set ID.",
+    "example": "1142",
+    "description": "The split {{link@Magic Card Market@https://www.cardmarket.com/en/Magic}} set ID if a set is printed in two sets. This ID represents the second set's ID.",
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/set.schema.json:125]: include referral

File: docs/.vuepress/src/schemas/set.schema.json [hunk 8]
```diff
@@ -100,22 +105,35 @@
...
+    "isManifestProperty": true,
+    "attributes": [
+      "optional"
+    ],
+    "introduced": "v5.0.2"
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/set.schema.json:130]: v5.1.0

File: docs/.vuepress/src/schemas/set.schema.json [hunk 9]
```diff
@@ -100,22 +105,35 @@
...
+  },
+  "mcmName": {
+    "type": "string",
+    "example": "\"Ravnica Allegiance\"",
+    "description": "The {{link@Magic Card Market@https://www.cardmarket.com/en/Magic}} set name.",
```
Comment (ZeldaZach) [inline docs/.vuepress/src/schemas/set.schema.json:135]: Referral URL

File: docs/.vuepress/theme/global-components/Documentation.vue [hunk 10]
```diff
@@ -110,17 +110,19 @@ export default {
...
-        let attr = this.schema[prop].attributes;
+      for (let prop in this.filteredSchema) {
+        let attr = this.filteredSchema[prop].attributes;
 
         if (attr) {
           count = count + 1;
         }
       }
 
+      console.log(count)
```
Comment (ZeldaZach) [inline docs/.vuepress/theme/global-components/Documentation.vue:124]: console?

File: docs/abstract-models/all-identifiers/README.md [hunk 11]
```diff
@@ -83,7 +83,6 @@ The All Identifiers data model describes a list of all cards organized by UUID.
         "text": "A spell that can’t be countered can still be targeted by spells or abilities that would counter it. The part of their effect that would counter the spell won’t do anything, but any other effects those spells or abilities may have will still happen, if applicable."
       }
     ],
-    "setCode": "POGW",
```
Comment (ZeldaZach) [inline docs/abstract-models/all-identifiers/README.md:86]: was this intentional?

Comment (staghouse) [reply docs/abstract-models/all-identifiers/README.md]: weird. added back

File: docs/mtggraphql/README.md [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+  }
+}
+```
+
+## HTTP Request
+
+You can use an HTTP request on the mtgjson.com GraphQL endpoint with the following format:
+
+```
+https://mtgjson.com/gql/v1?query={getCard(input:{name:"Phelddagrif"}){name,text,type}}
```
Comment (ZeldaZach) [inline docs/mtggraphql/README.md:75]: /graphql?query=...

**Pull request merged**